.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -74,5 +75,4 @@ We’ve also implemented loads of other improvements and fixed a heap of bugs.
 
 ```
 - [ ] Add one last update to the #product-team Slack channel and use the thread to lead an internal release retrospective covering what went well and areas to improve for next time.
-- [ ] Go to the [Issues tab](https://github.com/medic/cht-core/issues) and filter the issues with `is:issue label:"Affects: 4.x.x" ` , replace `4.x.x` with the previous version number. Add any open "known issues" from the prior release that were not fixed in this release. Done by adding the correct `Affects: 4.x.x` label.  
```

Comment:
I think we want keep this line - please add back

Reply:
oh - so you did - sorry I missed that !

Reply:
We definitely still need the line - has it been moved above?